task1_weatherman/main.py

```diff
@@ -0,0 +1,42 @@
+
+from weatherman import Weathermachine
+from filehandling import FileReader
+import argparse
+import sys
+
+
+
+folder_path = "weatherfiles"
+target_year = 2006
+target_month = 8
+
+weather_machine = Weathermachine() 
+file_reading = FileReader(folder_path)
+
+files_with_year_month = file_reading.get_files_by_year_month(target_year, target_month)
+
+def data_for_a_year():
```

Comment:
Let's add for months and other duration as well and move these methods into weathercalculator class.